azurerm/resource_arm_media_services_test.go

```diff
@@ -0,0 +1,298 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/services/mediaservices/mgmt/2018-07-01/media"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func init() {
+	resource.AddTestSweepers("azurerm_media_services", &resource.Sweeper{
+		Name: "azurerm_media_services",
+		F:    testSweepMediaServicesAccount,
+	})
+}
+
+func testSweepMediaServicesAccount(region string) error {
+	armClient, err := buildConfigForSweepers()
+	if err != nil {
+		return err
+	}
+
+	client := (*armClient).mediaServicesClient
+	ctx := (*armClient).StopContext
+
+	log.Printf("Retrieving the Media Services Accounts..")
+	results, err := client.List(ctx, "testrg")
+	if err != nil {
+		return fmt.Errorf("Error Listing on Media Services Accounts: %+v", err)
+	}
+
+	for _, account := range results.Values() {
+		if !shouldSweepAcceptanceTestResource(*account.Name, *account.Location, region) {
+			continue
+		}
+
+		resourceId, err := parseAzureResourceID(*account.ID)
+		if err != nil {
+			return err
+		}
+
+		resourceGroup := resourceId.ResourceGroup
+		name := resourceId.Path["mediaservices"]
+
+		log.Printf("Deleting Media Services Account '%s' in Resource Group '%s'", name, resourceGroup)
+		_, err = client.Delete(ctx, resourceGroup, name)
+		if err != nil {
```

Comment:
_Very minor_n but these two lines could be joined:
```go
		if _, err = client.Delete(ctx, resourceGroup, name); err != nil {
```

Reply:
Joined the two lines